bitstreamformats.md

```diff
@@ -0,0 +1,96 @@
+# Bitstream Formats Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**GET /api/core/bitstreamformats**   
+
+Provide access to the bitstream formats defined in the registry (DBMS based). It returns the list of existent metadata fields.
+
+Example: <https://dspace7.4science.cloud/dspace-spring-rest/#https://dspace7.4science.cloud/dspace-spring-rest/api/core/bitstreamformats>
+
+**POST /api/core/bitstreamformats**   
+
+Create a new bitstream format in the registry (DBMS based). Requires an admin account
+
+The JSON should be similar to:
+```json
+{
+  "shortDescription": "XML",
+  "description": "Extensible Markup Language",
+  "mimetype": "text/xml",
+  "supportLevel": "KNOWN",
+  "internal": false,
+  "extensions": [
+          "xml"
+  ],
+  "type": "bitstreamformat"
+}
+```
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Forbidden - if you are not authenticated
```

Comment:
we could specify here 400 for invalid supportLevel. I tend to prefer  400 in this case over 422 as it seems to me a syntatically error in the request format and not a semantically error but I have no a strong opinion on that so if you prefer 422 I will be fine also with that. We should just document what we expect to use.